<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyDecision2ConfigWidget</class>
 <widget class="QWidget" name="BuyDecision2ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>27</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QCheckBox" name="enabledCheckBox">
       <property name="text">
        <string>Buy stocks if the price has fallen and risen by more than</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="priceDiffDoubleSpinBox">
       <property name="suffix">
        <string notr="true"> %</string>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>and more than</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="amountOfTimesSpinBox">
       <property name="suffix">
        <string> times</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>in the last</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="durationSpinBox">
       <property name="suffix">
        <string> minutes</string>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
